<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductEditor</class>
 <widget class="QWidget" name="ProductEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Produk Baru</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>Informasi Umum</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1,2">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="generalGroup">
         <property name="title">
          <string>Informasi Utama</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,2,0,1,0,0">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="0" column="4">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Kode</string>
            </property>
            <property name="buddy">
             <cstring>idEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Jenis</string>
            </property>
            <property name="buddy">
             <cstring>typeComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="categoryComboBox"/>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="typeComboBox">
            <item>
             <property name="text">
              <string>Stok</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Non Stok</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jasa</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Kategori</string>
            </property>
            <property name="buddy">
             <cstring>categoryComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLineEdit" name="idEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Otomatis</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>&amp;Status</string>
            </property>
            <property name="buddy">
             <cstring>statusComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="statusComboBox">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Tidak Aktif</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Aktif</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLineEdit" name="nameEdit">
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Nama</string>
            </property>
            <property name="buddy">
             <cstring>nameEdit</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="uomGroup">
         <property name="title">
          <string>Satuan-satuan</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Nama Satuan Dasar</string>
            </property>
            <property name="buddy">
             <cstring>baseUomEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="baseUomEdit">
            <property name="maxLength">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QTableView" name="uomTableView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>20</number>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Satuan Lainnya:</string>
            </property>
            <property name="buddy">
             <cstring>uomTableView</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Modal</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Metode Penentuan Modal</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="costingMethodComboBox">
            <item>
             <property name="text">
              <string>Ditetapkan Manual</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Harga Beli Rata-rata</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Harga Beli Terakhir</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Harga Beli Manual</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="manualCostEdit">
            <property name="text">
             <string>0</string>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Harga Beli Rata-rata</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="averageCostEdit">
            <property name="text">
             <string>0</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Harga Beli Terakhir</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lastPurchaseCostEdit">
            <property name="text">
             <string>0</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>&amp;Harga Jual</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableView" name="priceTableView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>20</number>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>idEdit</tabstop>
  <tabstop>categoryComboBox</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>statusComboBox</tabstop>
  <tabstop>baseUomEdit</tabstop>
  <tabstop>uomTableView</tabstop>
  <tabstop>costingMethodComboBox</tabstop>
  <tabstop>manualCostEdit</tabstop>
  <tabstop>averageCostEdit</tabstop>
  <tabstop>lastPurchaseCostEdit</tabstop>
  <tabstop>priceTableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
